server/src/main/java/io/crate/analyze/GenericPropertiesConverter.java

```diff
@@ -89,6 +91,24 @@ static void settingsFromProperties(Settings.Builder builder,
                                        boolean setDefaults,
                                        Predicate<String> ignoreProperty,
                                        String invalidMessage) {
+        for (Map.Entry<String, Object> property: properties.properties().entrySet()) {
+            if (property.getValue() == null) {
+                if (property.getKey().equals(stripIndexPrefix(INDEX_REFRESH_INTERVAL_SETTING.getKey()))) {
+                    throw new IllegalArgumentException(
+                        String.format(
+                            Locale.ENGLISH,
+                            "Cannot set NULL to property %s. Use ALTER TABLE RESET to reset to default value.",
+                            property.getKey()
+                        )
+                    );
```

Comment:
Why the special handling for refresh? Reset can generally be used to reset settings to their default value?

Reply:
Was somehow focused only on refresh, thx for catching!|

Fixed in https://github.com/crate/crate/pull/13849/commits/742c1a5bba304edc7d4c82268b53a68904473ea1